strapi/gcp-deployment/package.json

```diff
@@ -8,7 +8,10 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "gcp-build": "strapi build"
+    "gcp-build": "strapi build",
+    "app-deploy": "gcloud app deploy app.yaml --project=ch-test-strapi --bucket=gs://ch-test-strapi-app-bucket",
+    "terraform-deploy-init": "terraform apply -var-file=\"secret.tfvars\" -target=google_storage_bucket.terraform_state",
```

Comment:
Wow, TIL about `--target`. Terraform has come a long way. My old school tf thinking makes me think that the terraform state should be in a separate tf config from the main infra, but this is probably fine.